tests/playwright/src/ai-lab-extension.spec.ts

```diff
@@ -113,11 +115,26 @@ test.describe.serial(`AI Lab extension installation and verification`, () => {
       }
       await playExpect(errorTab, `Error Tab was present with stackTrace: ${stackTrace}`).not.toBeVisible();
     });
+
     test(`Verify AI Lab extension is installed`, async ({ runner, page, navigationBar }) => {
       [page, webview] = await handleWebview(runner, page, navigationBar);
       aiLabPage = new AILabPage(page, webview);
       await aiLabPage.navigationBar.waitForLoad();
     });
+
+    test(`Enable GPU support`, async ({ page, navigationBar }) => {
+      test.skip(!AI_LAB_TEST_GPU_SUPPORT, 'GPU support is not enabled');
+      test.setTimeout(120_000);
+      await aiLabPage.enableGpuSupport();
```

Comment:
There should be specific section/describe with tests, that will verify:
* banner for gpu enablement is displayed by default when dashboard is opened first time
* GPU button from banner enables/checks the preferences entry
* Banner is disolved after this action (we will not check dont display anymore)
* describe will contain afterAll hook which will reset gpu to be disabled since it is default setup if something fails

Reply:
done